docs/users.rst

```diff
@@ -795,6 +802,70 @@ archive provided by cachito in the buildroot workdir.
   supporting Golang version is required. In other words, you should use Golang
   >= 1.13
 
+remote_sources
+~~~~~~~~~~~~~~
+Alternative to remote_source map that allows multiple remote sources.
+The usage of remote_sources has to be allowed in the OSBS Instance configuration. See :ref:`allow-multiple-remote-sources`.
+remote_sources map shares many of its keys with remote_source map but has two additional keys:
+
+\- name
+    Serves as a unique identifier for remote source.
+    It is a non-empty unique string containing only alphanumeric characters, underscore or dash.
+
+remote_source:
+    See :ref:`remote-source-keys`.
+
+
+container.yaml example with multiple remote sources:
+
+.. code-block:: yaml
+
+        remote_sources:
+        - name: cachito-pip-with-deps
+          remote_source:
+            repo: https://github.com/cachito-testing/cachito-pip-with-deps
+            ref: 56efa5f7eb4ff1b7ea1409dbad76f5bb378291e6
+            pkg_managers: [“pip”]
+        - name: cachito-gomod-with-deps
+          remote_source:
+            repo: https://github.com/cachito-testing/cachito-gomod-with-deps
+            ref: 21e42c6a62a23002408438d07169e2d7c76649c5
+            pkg_managers: [“gomod”]
+
+
+
+Once the `remote_sources` map described above is set in ``container.yaml``, you
+can now copy the upstream sources (with bundled dependencies) provided by
+cachito in your build image by adding::
+
+   COPY $REMOTE_SOURCES $REMOTE_SOURCES_DIR
+
+to your Dockerfile. This ``$REMOTE_SOURCES_DIR`` directory contains subdirectory for each remote source.
+You can access the source of an individual remote source at ``$REMOTE_SOURCES_DIR/{name}/app``.
+The dependencies can be correspondingly found at ``$REMOTE_SOURCES_DIR/{name}/deps``
+
+OSBS also creates ``$REMOTE_SOURCES_DIR/{name}/cachito.env`` bash script with exported
+environment variables received from cachito request (such as ``GOPATH``,
+``GOCACHE``) with the absolute path to the directory where the sources are
+expected to be. The Golang sources rely on the existence of the dependencies
+in this directory. OSBS adds these variables into Dockerfile as arguments
```

Comment:
this is confusing

osbs adds now for multiple remote sources as variables only : REMOTE_SOURCES, REMOTE_SOURCES_DIR

single remote source adds additionally : CACHITO_ENV_FILE
AND all variables from cachito

Reply:
updated